bitnami/kong/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kong
-version: 1.1.7
+version: 1.1.8
```

Comment:
Please bump the minor version instead, you're adding a new feature to the chart.

Reply:
updated to 1.2.7